src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -72,14 +72,14 @@
     <DebugSymbols>true</DebugSymbols>
     <Optimize Condition="'$(Optimize)' == '' and '$(Configuration)' == 'Debug'">false</Optimize>
     <Optimize Condition="'$(Optimize)' == '' and '$(Configuration)' == 'Checked'">true</Optimize>
-    <DebugType>full</DebugType>
+    <DebugType Condition="'$(DebugType)' == ''">full</DebugType>
```

Comment:
I think you can just delete these lines.

Reply:
Indeed this was my first version.  I put them back because they may be useful if we ever go back to MS-PDB (but that is a pretty low-probability event).  I am OK with removing them.